### PR TITLE
ispc: 1.18.1 -> 1.19.0

### PR DESCRIPTION
**Pull request by alexfmpe**
###### Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [x] aarch64-darwin
- [x] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [x] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [23.11 Release Notes (or backporting 23.05 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2305-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

**Review by athas — APPROVED** [review]

**Review by thoughtpolice — COMMENTED** [review]
Did you try upgrading to 1.20 as well, or only 1.19? This isn't a blocker, I was just curious.

**Comment by alexfmpe** [comment]
> Did you try upgrading to 1.20 as well, or only 1.19? This isn't a blocker, I was just curious.

Yeah but the build failed with some error I couldn't figure out at the time and I won't be at the computer until next week

File: pkgs/top-level/all-packages.nix [hunk 1]
```diff
@@ -21323,7 +21323,8 @@ with pkgs;
   isort = with python3Packages; toPythonApplication isort;
 
   ispc = callPackage ../development/compilers/ispc {
-    inherit (llvmPackages) stdenv;
+    inherit (llvmPackages_13) stdenv;
+    llvmPackages = llvmPackages_13;
```
[the remark this answers is not in the document]
Comment (alexfmpe) [reply pkgs/top-level/all-packages.nix:21327]: Well spotted. I won't be able to fix/test myself until next week. At most, I could accept a patch comment

**Comment by alexfmpe** [comment]
I've also gotten 1.20 to _build_ both on linux and mac, but turns out it seg faults when running on mac. Not sure why.
https://github.com/alexfmpe/nixpkgs/commit/be32a974ab3d7097904f698dbf915a5cdc1913f9

**Comment by alexfmpe** [comment]
Odd. It segfaults with llvm 15, but not with 14

**Review by athas — APPROVED** [review]
Cannot speak for macOS, though.

**Comment by Janik-Haag** [comment]
Result of `nixpkgs-review pr 235474` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>6 packages built:</summary>
  <ul>
    <li>blender</li>
    <li>blender-hip</li>
    <li>embree</li>
    <li>embree2</li>
    <li>ispc</li>
    <li>openimagedenoise</li>
  </ul>
</details>

**Review by Janik-Haag — APPROVED** [review]

**Pull request merged**